prime-router/metadata/valuesets/florida.valuesets

```diff
@@ -0,0 +1,55 @@
+- name: fl/student_teacher_other
+  system: LOCAL
+  values:
+      - display: Student
+        code: Student
+      - display: Teacher
+        code: Teacher
+      - display: Other (Faculty Member)
+        code: Other (Faculty Member)
+      - display: No
+        code: N
+      - display: Unknown
+        code: U
+
+- name: fl/abnormal_flag
+  system: LOCAL
+  values:
+      - display: Abnormal
+        code: A
+      - display: Normal
+        code: N
+
+- name: fl/congregate_care_setting
```

Comment:
This appears to be a standard type-of-institution / environment value in SNOMED.   I think this can just go straight into common.valuesets.

Reply:
I moved `congregate_care_setting` into common.valuesets and named it `site_of_care`.